### PR TITLE
Release 0.8.9

### PR DESCRIPTION
**Pull request by yatharthranjan**
- Added Ticwatch schemas and specifications
- Updated Google Places type information

### CODE REVIEW

**Review by this-Aditya — APPROVED** [review]
LGTM

File: commons/passive/google/google_places_info.avsc [hunk 1]
```diff
@@ -11,11 +11,15 @@
...
-        }], "doc": "Categorizing places based on their characteristics or attributes, this field represents the first type, if any, among the retrieved place categories.", "default": null
+        }], "doc": "Categorizing places based on their characteristics or attributes, This field is deprecated, as the Google Place.Type enum is deprecated. Instead, use the placeType field of string type.", "default": null
```
Comment (mpgxvii) [inline commons/passive/google/google_places_info.avsc:14]: If this is deprecated, do we still need to keep this?

File: commons/passive/google/google_places_info.avsc [hunk 2]
```diff
@@ -11,11 +11,15 @@
...
+    { "name": "type2", "type": ["null", "org.radarcns.passive.google.PlacesType"], "doc": "Categorizing places based on their characteristics or attributes, This field is deprecated, as the Google Place.Type enum is deprecated. Instead, use the placeType field of string type.", "default": null },
+    { "name": "type3", "type": ["null", "org.radarcns.passive.google.PlacesType"], "doc": "Categorizing places based on their characteristics or attributes, This field is deprecated, as the Google Place.Type enum is deprecated. Instead, use the placeType field of string type.", "default": null },
+    { "name": "type4", "type": ["null", "org.radarcns.passive.google.PlacesType"], "doc": "Categorizing places based on their characteristics or attributes, This field is deprecated, as the Google Place.Type enum is deprecated. Instead, use the placeType field of string type.", "default": null },
+    { "name": "placeType1", "type": ["null", "string"], "doc": "Categorizing places based on their characteristics or attributes, this field represents the first type, if any, among the retrieved place categories.", "default": null },
```
Comment (mpgxvii) [inline commons/passive/google/google_places_info.avsc:19]: Could we make the type an array of strings so we don't have to specify the place type number? Or are do we not want nested types?

File: commons/passive/google/google_places_info.avsc (hunk 1 above)
Comment (mpgxvii) [reply commons/passive/google/google_places_info.avsc:14]: Btw, can't we just do the enum conversion on our side or is it not a fixed set of values anymore?

Comment (this-Aditya) [reply commons/passive/google/google_places_info.avsc:14]: > If this is deprecated, do we still need to keep this?

As per [Yatharth's comment](https://github.com/RADAR-base/RADAR-Schemas/pull/362#discussion_r1523100375), we cannot delete existing fields in Avro schemas. Therefore, I have kept the field without removing it.

Comment (mpgxvii) [reply commons/passive/google/google_places_info.avsc:14]: Oh okay I see, sorry I missed that. What about creating a new version?

Comment (yatharthranjan) [reply commons/passive/google/google_places_info.avsc:14]: Yes this is for backwards compatibility and in line with the schema evolution guidelines. We can probably create a new version but not necessary IMO.

**Review by mpgxvii — APPROVED** [review]
LGTM.

**Comment by Bdegraaf1234** [comment]
I don't think these changes were ever merged back into dev, leading to some confusion on our end.

I merged them back now :)

**Pull request merged**